build/ci.go

```diff
@@ -345,6 +347,67 @@ func downloadSpecTestFixtures(csdb *build.ChecksumDB, cachedir string) string {
 	return filepath.Join(cachedir, base)
 }
 
+func hashFile(path string) (common.Hash, error) {
+	f, err := os.OpenFile(path, os.O_RDONLY, 0666)
+	if err != nil {
+		return common.Hash{}, err
+	}
+	contents, err := io.ReadAll(f)
+	if err != nil {
+		return common.Hash{}, err
+	}
+	return common.BytesToHash(crypto.Keccak256(contents)), nil
```

Comment:
Please use sha256, for two reasons:

- It is hardware-accelerated, while keccak256 is not.
- We should avoid depending on go-ethereum's crypto package in build/ci.go because compiling that package needs cgo. Generally best to try using stdlib as much as possible in build scripts.